DuckDuckGo/AutofillPixelReporter.swift

```diff
@@ -78,20 +83,29 @@ final class AutofillPixelReporter {
         firePixels(pixelsToFireFor(.fill))
     }
 
+    func checkIfOnboardedUser() {
```

Comment:
Moved this so that it can be triggered from app launch and app entering foreground